relayer/application-standard.ts

```diff
@@ -31,6 +31,9 @@ export interface StandardRelayerAppOpts extends RelayerAppOpts {
   privateKeys?: Partial<{
     [k in ChainId]: any[];
   }>;
+  tokenAddresses?: Partial<{
+    [k in ChainId]: any[];
```

Comment:
```
[k in ChainId]: string[];
```
or you could also use `TokenAddresses` exported on `wallet-management.ts`

Reply:
done